### PR TITLE
Add lv.martinsz.millionaire

### PR DESCRIPTION
**Pull request by martinszeltins**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build lv.martinsz.millionaire

**Comment by martinszeltins** [comment]
Looks like it failed not being able to resolve `gitlab.gnome.org`?

````
Cloning into 'blueprint-compiler'...
fatal: unable to access 'https://gitlab.gnome.org/jwestman/blueprint-compiler.git/': Could not resolve host: gitlab.gnome.org
````

**Comment by hfiguiere** [comment]
build is supposed to be performed offline, ie not conneting to the internet. Which is why this is failing.

File: lv.martinsz.millionaire.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "millionaire",
+            "builddir" : true,
```
Comment (TingPing) [inline lv.martinsz.millionaire.json:28]: You can remove this line. It's always true for `meson`.

File: lv.martinsz.millionaire.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
```
Comment (TingPing) [inline lv.martinsz.millionaire.json:23]: These seem to be a random copy paste list. None of them apply to your project as-is.

**Pull request closed without merging**